assets/models/system/microsoft-phi-2/spec.yaml

```diff
@@ -0,0 +1,29 @@
+$schema: https://azuremlschemas.azureedge.net/latest/model.schema.json
+name: microsoft-phi-2
+path: ./
+properties:
+  datasets: StackOverflow, Stackv1.2, CodeContests, gpt-3.5-turbo-0301
```

Comment:
Please take reference from OSS PRs related to movement, introduction and name change for few properties/tags. Please ignore in case you are planning to update this later on as part of OSS PR